packages/MJCore/src/generic/interfaces.ts

```diff
@@ -159,15 +170,66 @@ export class CompositeKey {
         });
     }
 
+    /**
+     * Loads the KeyValuePairs from a list of strings in the format "FieldName=Value"
```

Comment:
@JS-BC  you changed the format - it was previously FieldName|Value so that we could have a URL where it is `Field1|Value1||Field2|Value2||Field3|Value3` I did it this way to avoid using = in URL strings because = has meaning in that context. We don't want to change things like this around without significnat design discussion. I'm totally open to other formats, but we are making a change here that was not discussed

Reply:
The `LoadFromList` function is specific to the record dupe work where we parse the KeyValue pairs from a vector's metadata. The function to parse KeyValue pairs from a URL segment is `LoadFromURLSegment`